lib/chef/knife/opc_user_create.rb

```diff
@@ -0,0 +1,101 @@
+#
+# Author:: Steven Danna (<steve@chef.io>)
+# Copyright:: Copyright 2011-2016 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+require_relative "../mixin/root_rest"
+
+module Opc
+  class OpcUserCreate < Chef::Knife
+    category "CHEF ORGANIZATION MANAGEMENT"
+    banner "knife opc user create USERNAME FIRST_NAME [MIDDLE_NAME] LAST_NAME EMAIL PASSWORD"
```

Comment:
We already have a set of `knife user` commands that I think should cover most of the user-related commands in this PR.

Reply:
I'd like to see one or the other, but not both user commands. Since every chef server is a server with an org at this point we should just make any missing functionality part of the existing `knife user` operations.